devices/stm32/stm32f3-03-6_8.xml

```diff
@@ -133,46 +133,86 @@
       <channels instance="1">
         <channel position="1">
           <signal driver="adc" instance="1" name="dma"/>
-          <signal driver="tim" instance="17" name="ch1"/>
-          <signal driver="tim" instance="17" name="up"/>
+          <signal driver="tim" instance="17" name="ch1">
+            <remap position="12" mask="1" id="0"/>
+          </signal>
+          <signal driver="tim" instance="17" name="up">
+            <remap position="12" mask="1" id="0"/>
+          </signal>
```

Comment:
cc @rleh @chris-durand 

I've added these remappings manually from the RM, so we can discuss the formatting. I believe this is a compact and unique mapping, but I have to check this computationally.

This would require the dma driver module file to compute the conflicts somehow, but the information is all here.

Reply:
Note that similar to the STM32F1 remap, the bit position for the config registers is directly encoded here, to be independent of STs naming register conventions.